lib/node_modules/@stdlib/math/base/special/sec/test/test.js

```diff
@@ -0,0 +1,397 @@
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var PI = require( '@stdlib/constants/float64/pi' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var sec = require( '@stdlib/math/base/special/sec/lib' );
+
+
+// FIXTURES //
+
+var hugeNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/huge_negative.json' );
+var hugePositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/huge_positive.json' );
+var veryLargeNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/very_large_negative.json' );
+var veryLargePositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/very_large_positive.json' );
+var largeNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/large_negative.json' );
+var largePositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/large_positive.json' );
+var mediumNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/medium_negative.json' );
+var mediumPositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/medium_positive.json' );
+var smallNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/small_negative.json' );
+var smallPositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/small_positive.json' );
+var smaller = require( '@stdlib/math/base/special/sec/test/fixtures/julia/smaller.json' );
+var tinyNegative = require( '@stdlib/math/base/special/sec/test/fixtures/julia/tiny_negative.json' );
+var tinyPositive = require( '@stdlib/math/base/special/sec/test/fixtures/julia/tiny_positive.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.true( typeof sec, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the secant (huge negative values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	x = hugeNegative.x;
+	expected = hugeNegative.expected;
+
+	for (i = 0; i < x.length; i++) {
+		y = sec(x[i]);
+		console.log(y)
+		if (y === expected[i]) {
+			t.equal(y, expected[i], 'x: ' + x[i] + '. E: ' + expected[i]);
+		} else {
+			delta = abs(y - expected[i]);
+			tol = 1.4 * EPS * abs(expected[i]);
+			t.ok(delta <= tol, 'within tolerance. x: ' + x[i] + '. y: ' + y + '. E: ' + expected[i] + '. tol: ' + tol + '. Δ: ' + delta + '.');
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the secant (huge positive values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	x = hugePositive.x;
+	expected = hugePositive.expected;
+
+	for (i = 0; i < x.length; i++) {
+		y = sec(x[i]);
+		if (y === expected[i]) {
```

Comment:
Please honor the project's spacing conventions here and elsewhere throughout this PR.